src/components/poll/SubmitChoices.tsx

```diff
@@ -31,6 +44,16 @@ const SubmitChoices = (props: {
         };
         const submitChoiceResponse = await markChoices(voterArgs);
         if (submitChoiceResponse.statusCode === 201) {
+          if (isLoggedIn) {
+            const mailerArgs: MailerArgs = {
+              pollID,
+              senderEmailID,
+              pollTitle: pollFromDB.title,
+              receiverIDs: Array<string>(pollCreatorEmailID),
+              senderName: newVote.name,
+            };
+            sendPollResponse(mailerArgs, token);
+          }
           Router.reload();
```

Comment:
After commenting line 57 I took a screenshot of network logs and It shows the request is working fine 
![image](https://user-images.githubusercontent.com/54989497/111898980-6961d200-8a4f-11eb-98c0-4c78e1c95358.png)